<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgEnvironment</class>
 <widget class="QDialog" name="CQDlgEnvironment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Environment Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Colors / Fog</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QToolButton" name="qqBackgroundColorUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Background (up)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqBackgroundColorDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Background (down)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAmbientLightColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Ambient light</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqFogAdjust">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust fog parameters</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Wireless transmission visualization</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="qqShowWirelessEmissions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visualize wireless emissions</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowWirelessReceptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visualize wireless receptions</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqWirelessEmissionColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Set color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqWirelessReceptionColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Set color</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Calculation preprocessing</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum triangle size (absolute)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMaxTriangleSize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqSaveCalcStruct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Save operation also saves existing calculation structures</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum triangle size (relative)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMinRelTriangleSize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Various</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="qqShapeTexturesDisabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Shape textures disabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqUserInterfaceTexturesDisabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Custom user interface textures disabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqNextSaveIsDefinitive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Lock scene after next scene save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqCustomContactHandling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Custom collision/contact response</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqEditCustomContact">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Edit contact response</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqCleanUpHashNames">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clean-up object names</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqCleanUpGhosts">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clean-up ghosts</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Scene content acknowledgments / Info</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPlainTextEdit" name="qqAcknowledgments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqBackgroundColorUp</tabstop>
  <tabstop>qqBackgroundColorDown</tabstop>
  <tabstop>qqAmbientLightColor</tabstop>
  <tabstop>qqFogAdjust</tabstop>
  <tabstop>qqShowWirelessEmissions</tabstop>
  <tabstop>qqWirelessEmissionColor</tabstop>
  <tabstop>qqShowWirelessReceptions</tabstop>
  <tabstop>qqWirelessReceptionColor</tabstop>
  <tabstop>qqMaxTriangleSize</tabstop>
  <tabstop>qqSaveCalcStruct</tabstop>
  <tabstop>qqShapeTexturesDisabled</tabstop>
  <tabstop>qqUserInterfaceTexturesDisabled</tabstop>
  <tabstop>qqNextSaveIsDefinitive</tabstop>
  <tabstop>qqCustomContactHandling</tabstop>
  <tabstop>qqEditCustomContact</tabstop>
  <tabstop>qqCleanUpHashNames</tabstop>
  <tabstop>qqAcknowledgments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
